admin/schedule-form-excludes.php

```diff
@@ -298,12 +298,15 @@
 							$size = $site_size->filesize( $file );
 
 							if ( false !== $size ) :
-								$size          = $size ?: '0';
-								$excluded_size = $excluded_site_size->filesize( $file ) ?: '0'; ?>
+								$size          = $size ?: 0;
+								$excluded_size = $excluded_site_size->filesize( $file ) ?: 0;
```

Comment:
@willmot currently excluded files display size `3 of 3 KB` for example, due to this line. If you follow the function call `filesize()` you can see that for files it returns their size regardless if they are excluded or not, ref: https://github.com/humanmade/backupwordpress/blob/master/classes/class-site-size.php#L184

However, the function for calculating `directory_filesize()` in the same class is aware of excluded files - so that directory size is calculated without excluded files sizes.

Question: should we improve the `filesize()` in the `Site_Size` class to make it aware of excluded files? I.e. if the file is excluded then return `0`, otherwise the file size. I think that the class it the right place for it, as calculating sizes is its purpose. However, I wanted to double check.